editor/layouts/layout-styles.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Styles'
+description: 'Layouts can have styles applied to directly them'
+---
+
+import { YouTube } from '/snippets/youtube.mdx'
+
+Though Layouts behave as containers for other Rive objects, Layouts themselves can contain styles including:
+
+- Fills
+- Strokes
+- Feathers
+- Blend Modes
+- Corner Radius
+
+<Info>For additional information, see [Fill and Stroke](editor/fundamentals/fill-and-stroke) and [Fill and Stroke Tips](editor/fundamentals/fill-and-stroke-tips).</Info>
```

Comment:
Looks like both these links are broken. What might be missing is a `/` at the start of the path